browser/ui/views/brave_actions/brave_shields_action_view.cc

```diff
@@ -195,8 +196,8 @@ void BraveShieldsActionView::ButtonPressed() {
 bool BraveShieldsActionView::SchemeIsLocal(GURL url) {
   return url.SchemeIs(url::kAboutScheme) || url.SchemeIs(url::kBlobScheme) ||
          url.SchemeIs(url::kDataScheme) ||
-         url.SchemeIs(url::kFileSystemScheme) ||
-         url.SchemeIs(content::kChromeUIScheme);
+         url.SchemeIs(url::kFileSystemScheme) || url.SchemeIs(kMagnetScheme) ||
+         url.SchemeIs(kBraveUIScheme) || url.SchemeIs(content::kChromeUIScheme);
 }
```

Comment:
This is fine as incremental, but I wonder if it's better for the shields system itself should report back on whether shields is "useable" for the particular url (or WebContents). Since it must have this logic to decide whether to do shields-y stuff at a place more core to shields. It would also be re-usable for android UI.